synthea-hiv/uploader/bundle.py

```diff
@@ -28,7 +28,7 @@
 class Bundle:
   """Object to be uploaded to FHIR server."""
 
-  def __init__(self, file_name: pathlib.PosixPath, bundle_dict: Dict[str, str]):
+  def __init__(self, file_name: pathlib.PosixPath, bundle_dict: Dict[str, Any]):
```

Comment:
Is there a reason we prefer to use `Any` over `str`?

Reply:
Yeah, that was an IDE type-checker warning that I fixed. `str` is incorrect in this case as the range of elements in `bundle_dict` map is not limited to `str`s only.